index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio setup</title>
+  <link rel="stylesheet" href="./style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <header>
+    <div class="logo">
+      <img  src="./images/image_geometry_1.png" alt="logo image">
+   </div>
+    <nav>
+      <div class="menu">
+           <i class="fa fa-bars"></i>
+      </div>
+    </nav>
+  </header>
+  <main>
+  
+    <section class="headline"> 
+        <h1 class="myname">Charlie 
+        Giorgania
+        </h1> 
+        <p  class="headline-p">Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work. <br> <br>
+  
+          If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+      <button class="headline-btn">
+        Start collaboration
+      </button>
```

Comment:
You have done very well implementing this project without any serious issues with your code. Well done. 

- [ ] I noticed a little issue with the `Start Collaboration`, it has a gray color border that should be removed. This border is a default style of buttons. Kindly add `border: none;` to the `.headline-btn` class. 

Your button | Figma Button
---- | ----
![image](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/84156714/5ad4d128-4608-40d7-94c9-3a3bc1747238) | ![image](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/84156714/811bf47d-80e2-461e-89bf-fd9b7a9c004c)